### DISCUSSION TITLE
Help on Building Program

### DISCUSSION THREAD

**Discussion by bernard-grech** [NetworkingDevs/NS3-GUI-HELPER#22] [Q&A] [answered]
How can I build the Program? I am not seeing any NS3GUIHelper.java file.

**Comment by HenilMistry** [comment, marked as answer]
## Hello!
It's been so long that I haven't updated the ReadMe.md file so, The instructions that you might be following was for the very first release but not to worry! 

1. The first thing, That the main branch contains the code for the latest release from now onwards. 
2. Then second thing, if you want to build the program, you need to use any IDE like mentioned in that ReadMe.md file which is namely IntelliJ IDEA (Recommended!!)
3. If you're planning to contribute so, you can clone the main branch and then open it with any IDE. (Recommended Strongly: IntelliJ IDEA)
4. The last and very important, which might save your time, in case, if you just want to run the program and explore it, which is, to download the .jar file from the latest release. For that you can checkout [GitHub Releases](https://github.com/NetworkingDevs/NS3-GUI-HELPER/releases) section or [GitHub Pages](https://networkingdevs.github.io/NS3-GUI-HELPER/releases.html).
5. After downloading the file, you can just run that .jar file to get started!

Hope this helps you out, if not, you can ping in the discussion at any time.
